### ISSUE TITLE
[Crum] Complement Crum with Data from Westendorf

### ISSUE THREAD

**Issue by pishoyg** [pishoyg/coptic#121]
We have access to literature that Crum didn't have access to when he wrote his dictionary. While Crum remains the standard reference dictionary for Coptic, more recent dictionaries (including some from [this list](https://github.com/pishoyg/coptic/issues/411)) have expanded it.

Milan has added many entries from the Nag Hammadi Library, and some from other sources.

This issue asks to complement the database with words unknown at Crum's time but known today.

Note:
1. This work could require a substantial investment.
2. Its value is less evident given that such complements barely have any Bohairic, and are of mere interest to academia.
3. Its value is further reduced by #411, as well as the presence of the KELLIA database in our collection.

We may therefore abandon it in favor of other pursuits.

**NOTE:** There is a small error in the comments below. #9 doesn't concern itself with this issue, but rather with fixing typos in the data, i.e. entries present in Crum but absent from the Marcion database. Such entries concern both #9 and #508, but are more likely to be resolved in #508.

**Comment by pishoyg** [comment]
Abandoned for the reasons explained [here](#issue-2447109376)!

**Comment by pishoyg** [comment]
A future version of the [Comprehensive Coptic Lexicon](https://refubium.fu-berlin.de/handle/fub188/27813) is inevitable, and we have no capacity for it, but we may consider our digitization of Crum to be a great enabler for such an endeavour.

**Comment by pishoyg** [comment]
This doesn't help much with the goal of making Coptic more **learnable**.

**Comment by pishoyg** [comment]
#167 is not a strict blocker, but would be nice to have before we start expanding our dataset.

**Comment by pishoyg** [comment]
Consult with KELLIA before proceeding with this work. Don't do something that they're already doing.

**Comment by pishoyg** [comment]
Closing, provided that we do #305.

If we can't do #305, let's reopen this.

**Comment by remnqymi** [comment]
**NOTE**: #6 would enable us, not just to expand CDO, but also to expand Marcion. (_Edit_: #774!)

**NOTE**: As for #305, the upcoming update to the [BBAW](https://www.bbaw.de/)'s [Lexicon of Coptic Egyptian](https://edoc.bbaw.de/opus4-bbaw/frontdoor/index/index/docId/5400) (expected some time in 2028) seems to come from the databases compiled by Wolf-Peter Funk. No systematic adoption of the words available in Crum is in sight, and funding seems unavailable for the project, which means that the Lexicon of Coptic Egyptian may continue to be lacking in content in the future.

**NOTE**: How complete is Westendorf? Does it suffice to rely on Westendorf for our expansion? Or is there something else that we should incorporate?
